pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -788,3 +817,18 @@ func removeCryptoKeyVersion(kmsKey string) string {
 	}
 	return kmsKey
 }
+
+// encodeDiskTags encodes requested volume tags into JSON string, as GCE does
+// not support tags on GCE PDs and we use Description field as fallback.
```

Comment:
What about labels?
https://cloud.google.com/compute/docs/reference/rest/v1/disks/setLabels

Reply:
There is another PR adding labels (https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/566). My plan is to merge this, and poke that PR to incorporate these as labels as well.